ang/crmFunding/application/defaultApplicationEditor.directive.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2023 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+'use strict';
+
+fundingModule.directive('fundingDefaultApplicationEditor', function() {
+  return {
+    restrict: 'AE',
```

Comment:
how to know to use `AE` (short for ApplicationEditor?)

Reply:
See above.